lib/interface_quda.cpp

```diff
@@ -811,8 +795,7 @@ void loadGaugeQuda(void *h_gauge, QudaGaugeParam *param)
 
   if (extendedGaugeResident) {
     // updated the resident gauge field if needed
-    const int *R_ = extendedGaugeResident->R();
-    const int R[] = { R_[0], R_[1], R_[2], R_[3] };
+    const int R[] = {commDimPartitioned(0), commDimPartitioned(1), commDimPartitioned(2), commDimPartitioned(3)};
```

Comment:
This line is puzzling me.  This will give an `R` array with values of 0 or 1, but we only really support even numbers here (something that should be fied).  Why is this line changed like this?